.github/workflows/CODEOWNERS

```
@@ -0,0 +1,4 @@
+# Syntax for this file at https://help.github.com/articles/about-codeowners/
+
+*       @conda/build-tools
+*       @conda/conda-maintainers
```

This afaik has to be one line ... as later lines replace lines from before if matching the same things:

```suggestion
*       @conda/build-tools @conda/conda-maintainers
```

---

Thanks, good catch!